sentry_sdk/integrations/_asgi_common.py

```diff
@@ -15,6 +15,19 @@
     from sentry_sdk.utils import AnnotatedValue
 
 
+DEFAULT_HTTP_METHODS_TO_CAPTURE = (
```

Comment:
Should we maybe just import this from `_wsgi_common` here? Or directly use the `_wsgi_common` constant in the integrations?